Vagrantfile

```diff
@@ -22,8 +22,9 @@ Vagrant.require_version ">= 2.0.0"
 if ARGV.first == "up" && ENV['CILIUM_SCRIPT'] != 'true'
     raise Vagrant::Errors::VagrantError.new, <<END
 Calling 'vagrant up' directly is not supported.  Instead, please run the following:
-  export NWORKERS=n
-  ./contrib/vagrant/start.sh
+$ ./contrib/vagrant/start.sh
+
+See the development guide for available configuration options.
```

Comment:
Add a link to the guide - https://docs.cilium.io/en/latest/contributing/development/dev_setup/?